<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Poppins;
}

QWidget {
	background-color: white;
}</string>
  </property>
  <widget class="QLabel" name="baterryLevel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>11</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 0,
    stop:0 rgba(102, 164, 181, 255),
	stop:0.5 rgba(102, 164, 181, 255), 
	stop:0.51 rgba(38, 102, 118, 255), 
	stop:1 rgba(38, 102, 118, 255)
);
border-radius: 12px;
color: white;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="chargeIndicator">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="border">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>480</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(199, 199, 199);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timeLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #266676;
font-weight: bold;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>10:34</string>
   </property>
  </widget>
  <widget class="QLabel" name="networkLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font-size: 14pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
